Mathlib/Probability/Moments.lean

```diff
@@ -333,4 +344,363 @@ theorem measure_le_le_exp_cgf [IsFiniteMeasure μ] (ε : ℝ) (ht : t ≤ 0)
 
 end MomentGeneratingFunction
 
+section GeneratingFunctionDerivatives
+
+lemma aemeasurable_expt (X : Ω → ℝ) (t : ℝ) (hX : AEMeasurable X μ) :
+    AEStronglyMeasurable (fun ω ↦ rexp (t * (X ω))) μ :=
+  aestronglyMeasurable_iff_aemeasurable.mpr <| measurable_exp.comp_aemeasurable' (hX.const_mul t)
+
+lemma integrable_expt [IsFiniteMeasure μ] (X : Ω → ℝ) (t b : ℝ) (ht : t > 0)
+    (hX : AEMeasurable X μ) (hb : ∀ᵐ ω ∂μ, X ω ≤ b) :
+    Integrable (fun ω ↦ exp (t * (X ω))) μ := by
```

Comment:
You could have a much shorter proof: this is a bounded function, hence integrable.